src/main/java/seedu/taassist/logic/parser/ParserUtil.java

```diff
@@ -140,4 +140,12 @@ public static Set<ModuleClass> parseModuleClasses(Collection<String> moduleClass
         }
         return moduleClassSet;
     }
+
+    /**
+     * Returns true if the {@code prefix} does not contain empty {@code Optional} values in the given
+     * {@code ArgumentMultimap}.
+     */
+    public static boolean isPrefixPresent(ArgumentMultimap argumentMultimap, Prefix prefix) {
```

Comment:
Currently this class is only parsing strings. How about moving this function to `ArgumentMultimap`?

Reply:
Nice suggestion. In that case, I made it non-static and allowed checking of multiple prefixes